.github/workflows/build-storybook.yml

```
@@ -5,10 +5,17 @@ on:
     branches:
       - develop
       - master
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+      - synchronize
 
 jobs:
   build-storybook:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.action != 'closed'
```

The `github.event_name != 'pull_request'` check here is a bit confusing because we do want it to run for PRs, just not when they're closed of course 😄 

How about this?
```suggestion
    if: ! ( github.event_name == 'pull_request' && github.event.action == 'closed' )
```

This makes it closer to the opposite logic below used by `remove-storybook` as well.

---

Updated.